backend/cms/templates/pages/tree_row.html

```diff
@@ -5,25 +5,22 @@
         </span>
     </td>
     <td>
-        <a href="{% url 'edit_page' page_translation_id=node.id site_slug=site.slug %}" class="block py-3 px-2 text-grey-darkest">
-            {{ node }}
+        <a href="{% url 'edit_page' page_id=node.id site_slug=site.slug language_code=language.code %}" class="block py-3 px-2 text-grey-darkest">
+	        {% for page_translation in node.page_translations.all %}
+		        {% if page_translation.language == language %}
+			        {{ page_translation.title }}
+		        {% endif %}
+	        {% endfor %}
```

Comment:
We should probably write a custom template filter to get the title in the current language.
Or has anybody a better way of solving this?